change/@fluentui-web-components-86d45358-8f3e-454f-92b1-96b3ea94e9a0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "fix(web-components): remove final dependencies on fast-foundation",
+  "packageName": "@fluentui/web-components",
+  "email": "=",
```

Comment:
Should this be set?

Reply:
I don't think it matters - I'll look and see why it's "not" setting.